### PR TITLE
Highlight affected slices for filter change in dashboard view

### PR DESCRIPTION
**Pull request by vera-liu**
Issue: [https://github.com/airbnb/caravel/issues/659](url)
Done:
- When user adds/deletes a filter in dashboard, affected slices will
  have their header highlighted for 2 seconds

giphy 2

needs-review @ascott @mistercrunch

### CODE REVIEW

**Comment by ascott** [comment]
i wonder if a more subtle/fadein-out highlight could work here. i find the yellow header a little too jarring. maybe the highlight could go around the whole panel.

here's an idea, we can tweak the transition timing and color as needed. (the gif makes it look a little wonky. in the browser it's smoother)

panel-highlight

here's an example of the css needed:

http://codepen.io/ascott/pen/RGOyEW

**Comment by vera-liu** [comment]
That looks so neat! Thank you @ascott !

File: caravel/assets/stylesheets/dashboard.css [hunk 1]
```diff
@@ -81,6 +81,11 @@ div.widget .chart-controls {
 .slice-grid div.separator.widget h1,h2,h3,h4 {
   margin-top: 0px;
 }
+
```
Comment (ascott) [inline caravel/assets/stylesheets/dashboard.css]: we need to add css to the slice container so we get the nice transition too

```css
  box-shadow: 0px 0px 20px 5px rgba(0,0,0,0);      
  transition: box-shadow 1s ease-in;
```

File: caravel/assets/javascripts/dashboard/Dashboard.jsx [hunk 2]
```diff
@@ -201,6 +201,11 @@ function dashboardContainer(dashboardData) {
       this.slices.forEach(function (slice) {
         if (slice.data.slice_id !== sliceId && immune.indexOf(slice.data.slice_id) === -1) {
           slice.render();
+          const sliceSeletor = $(`#${slice.data.token}-cell`);
+          sliceSeletor.addClass('slice-highlight');
+          setTimeout(function () {
+            sliceSeletor.removeClass('slice-highlight');
+          }, 2000);
```
Comment (ascott) [inline caravel/assets/javascripts/dashboard/Dashboard.jsx]: with the transition timing, i might change this to `1200`ms

**Comment by ascott** [comment]
awesome! could upload a revised gif with the transition?

**Comment by ascott** [comment]
LGTM! thanks @vera-liu!

**Pull request merged**